drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -231,6 +242,13 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       report_age_date = [enrollment.EntryDate, report.filter.start].max
       report_item.age = enrollment.client.age_on(report_age_date)
       report_item.ch_at_entry = enrollment.chronically_homeless_at_start?
+      report_item.percent_ami = enrollment.PercentAMI
+      report_item.vamc_station = enrollment.VAMCStation
+      report_item.veteran = client.veteran?
```

Comment:
pretty sure `report_item.veteran` is an integer, so probably:
```suggestion
      report_item.veteran = client.veteran_status
```